packages/graph/.gitignore

```diff
@@ -0,0 +1,2 @@
+target/
+Cargo.lock
```

Comment:
I'm inclined to think we'll quickly undo this because we want a binary for the graph, but makes sense as is.

Reply:
I removed it as I added a dummy binary to the PR: 4cbcfedf11b5fb5e130746b84cea3564671d2017